plugin.xml

```diff
@@ -55,6 +55,7 @@
     <platform name="ios">
         <!-- required background modes:  App registers for location updates -->
         <hook type="after_plugin_install" src="hooks/add_swift_support.js"/>
```

Comment:
Is there a reason to keep the after_plugin_install hook here if were doing this on after_platform_add?